Source/textdat.cpp

```diff
@@ -501,161 +502,161 @@ const TextDataStruct alltext[] = {
 	    true, PS_NARATR4 },
-	{ "In Spiritu Sanctum. |", true, PS_WARR54 },
-	{ "Praedictum Otium. |", true, PS_WARR55 },
-	{ "Efficio Obitus Ut Inimicus. |", true, PS_WARR56 },
-	{ "In Spiritu Sanctum. |", true, PS_MONK54 },
-	{ "Praedictum Otium. |", true, PS_MONK55 },
-	{ "Efficio Obitus Ut Inimicus. |", true, PS_MONK56 },
-	{ "In Spiritu Sanctum. |", true, PS_MAGE54 },
-	{ "Praedictum Otium. |", true, PS_MAGE55 },
-	{ "Efficio Obitus Ut Inimicus. |", true, PS_MAGE56 },
-	{ "In Spiritu Sanctum. |", true, PS_ROGUE54 },
-	{ "Praedictum Otium. |", true, PS_ROGUE55 },
-	{ "Efficio Obitus Ut Inimicus. |", true, PS_ROGUE56 },
-	{ "In Spiritu Sanctum. |", true, PS_ROGUE54 },
-	{ "Praedictum Otium. |", true, PS_ROGUE55 },
-	{ "Efficio Obitus Ut Inimicus. |", true, PS_ROGUE56 },
+	{ N_("In Spiritu Sanctum. |"), true, PS_WARR54 },
+	{ N_("Praedictum Otium. |"), true, PS_WARR55 },
+	{ N_("Efficio Obitus Ut Inimicus. |"), true, PS_WARR56 },
+	{ N_("In Spiritu Sanctum. |"), true, PS_MONK54 },
+	{ N_("Praedictum Otium. |"), true, PS_MONK55 },
+	{ N_("Efficio Obitus Ut Inimicus. |"), true, PS_MONK56 },
+	{ N_("In Spiritu Sanctum. |"), true, PS_MAGE54 },
+	{ N_("Praedictum Otium. |"), true, PS_MAGE55 },
+	{ N_("Efficio Obitus Ut Inimicus. |"), true, PS_MAGE56 },
+	{ N_("In Spiritu Sanctum. |"), true, PS_ROGUE54 },
+	{ N_("Praedictum Otium. |"), true, PS_ROGUE55 },
+	{ N_("Efficio Obitus Ut Inimicus. |"), true, PS_ROGUE56 },
+	{ N_("In Spiritu Sanctum. |"), true, PS_ROGUE54 },
+	{ N_("Praedictum Otium. |"), true, PS_ROGUE55 },
+	{ N_("Efficio Obitus Ut Inimicus. |"), true, PS_ROGUE56 },
 	/**
-	{ "Please help....! I barely escaped from....... The Butcher...! He killed.... My wife... My children! I beg of you...... In the name of God......... Avenge them......... |", true, TSFX_DEADGUY },
-	{ "*retching* Listen, listen. I don't even like worms. Don't tell me about yer worms! I don't want yer worms! Nope, no thanks! *slight hiccup/gag* No worms for me.", true, TSFX_DRUNK5 },
+	{ N_"Please help....! I barely escaped from....... The Butcher...! He killed.... My wife... My children! I beg of you...... In the name of God......... Avenge them......... |"), true, TSFX_DEADGUY },
+	{ N_("*retching* Listen, listen. I don't even like worms. Don't tell me about yer worms! I don't want yer worms! Nope, no thanks! *slight hiccup/gag* No worms for me."), true, TSFX_DRUNK5 },
 	{ "*SNORE*", true, TSFX_DRUNK6 },

```

Comment:
oh look there is more
```suggestion
	{ _("*SNORE*"), true, TSFX_DRUNK6 },
```
and more below this one...

Reply:
... I'm not going to point out ever one of the renaming 125 messages missing from this point.